src/app/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="header">
+    <div class="header-content container">
+        <div class="header-logo-label">
+            <div class="header-logo"><img alt={{img_text}} src="../../../assets/icons/logo.svg"></div>
```

Comment:
```suggestion
            <div class="header-logo"><img [alt]="img_text" src="../../../assets/icons/logo.svg"></div>
```

Introduce a class property that hold the image path

Reply:
i have introduced a class property named imageUrl for the purpose vaia.